lib/cron/redshift-reaper.ts

```diff
@@ -9,7 +9,6 @@ const TABLES_TO_CLEAN = [
   'unifiedroutingresponses',
   'rfqrequests',
   'rfqresponses',
-  'archivedorders',
```

Comment:
are you removing archivedOrders from this so we don't delete those?

Reply:
realized archivedOrders don't have a timestamp field